enterprise/server/remote_execution/containers/firecracker/firecracker.go

```diff
@@ -621,6 +627,7 @@ func NewContainer(ctx context.Context, env environment.Env, task *repb.Execution
 		vmLog:            vmLog,
 		cancelVmCtx:      func(err error) {},
 	}
+	c.vmExec.once = &sync.Once{}
```

Comment:
does the connection still work when calling Pause() then Unpause() on the same instance, or do we need to also reset this in Pause()? (I think we don't reuse an instance after calling Pause() in practice, but might still have some tests that do)